src/material/chips/chip-option.spec.ts

```diff
@@ -331,6 +355,64 @@ describe('MDC-based Option Chips', () => {
 
         expect(optionElementDescription).toMatch(/option description/i);
       });
+
+      it('should display checkmark graphic by default', () => {
+        expect(
```

Comment:
Same note as above: this assertion can be removed since it's already covered by the DOM assertions.